.github/workflows/main.yml

```diff
@@ -24,6 +24,8 @@ jobs:
     container:
       image: jf002/infinitime-build
     steps:
+      - name: Workqround permission issues
```

Comment:
small typo fix
```suggestion
      - name: Workaround permission issues
```

Reply:
It's weird that Github allows reviews on older commit without any notification :p